sycl/plugins/unified_runtime/CMakeLists.txt

```diff
@@ -0,0 +1,93 @@
+# PI Unified Runtime plugin library
+#
+if (NOT DEFINED UNIFIED_RUNTIME_INCLUDE_DIR)
+	message(STATUS "Download Unified Runtime headers from github.com")
+  set(UNIFIED_RUNTIME_SOURCE_DIR "${CMAKE_CURRENT_BINARY_DIR}/unified_runtime")
+  if (NOT DEFINED SYCL_EP_UNIFIED_RUNTIME_SKIP_AUTO_UPDATE)
+	  set(SYCL_EP_UNIFIED_RUNTIME_SKIP_AUTO_UPDATE ${SYCL_EXTERNAL_PROJECTS_SKIP_AUTO_UPDATE})
+  endif()
+
+  set(UNIFIED_RUNTIME_SOURCE_VERSION main)
```

Comment:
Did you consider using submodules to achieve the same thing here? 

I'm weary of cloning the main development branch of the header repo by default since any API breaking changes in the headers repo could cause builds to break in this repo without anything having changed. By using submodules we could tie the header version in LLVM to a specific known working version. 

Obviously submodules have their draw backs too, and it would mean manually bumping the headers in this repo each time there is a change, which is perhaps not as convenient as the approach you have now.

Reply:
I never used submodules before hence can't tell if they are better here.
Note, however, that this code fetches specific tag ("main" now, but can be any stable version tag when/if needed).
Are there any other advantages of using submodules?
Can we add a TODO comment to research this further and not block this PR?

Reply:
I've added a TODO comment about exploring this further (I expect whole cmake machinery to be updated too).